Samples/Cordova/ExampleProject/platforms/android/app/src/main/assets/www/js/index.js

```diff
@@ -167,104 +157,105 @@ onDeviceReady: function() {
     //      }
```

Comment:
enhance ui by creating seperate callers for each function (similar to native sdk sample)